lib/fetchJSON.js

```diff
@@ -0,0 +1,15 @@
+var fs = require('fs');
+var Q = require('q');
+
+/** Reads from the configuration file and returns data
```

Comment:
From a stylistic perspective, the description should go on the next line:

```javascript
/**
 * Read from the configuration file and returns data
```

This finding applies to multiple comment blocks